### DISCUSSION TITLE
Build the Verilator model of Ariane by using the Makefile

### DISCUSSION THREAD

**Discussion by 2536786901** [openhwgroup/cva6#648] [Q&A]
Hi All
When I use the make verilate command, the error is reported as follows
![0fb34bcc38827dde42f9ef2259f992d](https://user-images.githubusercontent.com/58470681/114292346-6e4efa00-9ac0-11eb-9b1a-82a377be52ab.jpg)
How to solve this problem？

**Comment by jeremybennett** [comment]
@2536786901 What version of Verilator are you using?

**Reply by 2536786901** [reply]
I just installed it recently, it should be the latest version
verilator/focal,now 4.028-1 amd64 [installed]

**Comment by jeremybennett** [comment]
@2536786901 OK - that's ruled out the obvious cause. I'll need to leave it to the hardware guys to advise (I'm a tools engineer).

**Reply by 2536786901** [reply]
thank you very much
